airflow/executors/kubernetes_executor.py

```diff
@@ -601,18 +629,19 @@ def terminate(self):
 
 class KubernetesExecutor(BaseExecutor, LoggingMixin):
     """Executor for Kubernetes"""
+
     def __init__(self):
-        self.kube_config = KubeConfig()
-        self.task_queue = None
-        self.result_queue = None
-        self.kube_scheduler = None
-        self.kube_client = None
-        self.worker_uuid = None
+        self.kube_config: KubeConfig = KubeConfig()
```

Comment:
```suggestion
        self.kube_config = KubeConfig()
```

don't need this one (but do for the rest of the ones in this fn)

Reply:
Yeah.